### DISCUSSION TITLE
How can I spinn the another container of opensearch.

### DISCUSSION THREAD

**Discussion by agauttam** [utmstack/UTMStack#687] [Q&A] [answered]
My opensearch container RAM utilization is 100% I want to spinn another container

**Comment by c3s4rfred** [comment]
Hi @agauttam, we don't have clusterization over the opensearch in UTMStack yet, we are working to do it in some of the internal components, you must follow the system requirements for now, to have the spected performance. Best regards.

**Reply by agauttam** [reply]
I am already using 64 GB of RAM only for 75 devices, that is much more than suggested hardware requirement.
Do you have any suggestion to fix this issue.

Regards,
Arun

**Reply by agauttam** [reply]
Is there anyway to run redundant system to share the load.

Regards,
Arun

**Reply by agauttam** [reply]
When I spinn the installer to increase the memory for opensearch container, Its start consuming more memory.
for example before spanning 8GB out of 10.7 and after spinn 12 GB out of 21.9. Is this the right behavior.

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @agauttam our new installer automatically distributes the available memory up to 256 GB between the containers with a priority order and amount, to get the best use of your memory, and some of the ram memory is used for the operating system. So, yes, it is the right behavior.

**Reply by agauttam** [reply]
Hi Freddy,
Thank you for the update.

I customized the memory allocation in the installer, and it fixed the issue.

Regards,
Arun
